src/rev_ai/streamingclient.py

```diff
@@ -62,19 +62,24 @@ def __init__(self,
         self.on_connected = on_connected
         self.client = websocket.WebSocket(enable_multithread=True)
 
-    def start(self, generator, metadata=None):
+    def start(self, generator, custom_vocabulary_id=None, metadata=None):
```

Comment:
Ordering of parameters matters if the parameters are not named. You should add to the end of the parameter list otherwise this is a breaking change. @kbridbur Is this correct?

Reply:
You're correct, anyone who has
```
client.start(my_generator, my_metadata)
```
Will have broken code if we release this

Reply:
You're right. Will change!